config.xml

```diff
@@ -172,13 +172,16 @@
   <doc cat="sle-rt" doc="SLE-RT-shielding"  branches="main SLERT15SP1 SLERT15 SLERT12SP4 SLERT12SP3">Shielding Linux Resources</doc>
   <doc cat="sle-rt" doc="SLE-RT-virtguide"  branches="main SLERT15SP1 SLERT15 SLERT12SP4 SLERT12SP3">Virtualization Guide</doc>
 
+  <doc cat="sles-sap" doc="SLES-SAP-installation" branches="main">Quick Start (newer)</doc>
+  <doc cat="sles-sap" doc="SLES-SAP-automation" branches="main">SAP Automation (newer)</doc>
+  <doc cat="sles-sap" doc="SLES-SAP-monitoring" branches="main">SAP Monitoring (newer)</doc>
 
   <doc cat="sles-sap" doc="SLES-SAP-guide" branches="main 15_SP2">Guide (newer)</doc>
-  <doc cat="sles-sap" doc="SLES-SAP-quick" branches="main 15_SP2">Quick Start (newer)</doc>
+  <doc cat="sles-sap" doc="SLES-SAP-quick" branches="15_SP2">Quick Start (newer)</doc>
```

Comment:
Unfortunately, now we have `Quick Start (newer)` listed twice. This is not an issue as far as page generation goes but may be confusing to users. I also don't know which of the two entries will sort first. Incidentally, `Quick Start (newish)` would sort correctly, so I'd just suggest that in absence of better ideas:

```suggestion
  <doc cat="sles-sap" doc="SLES-SAP-quick" branches="15_SP2">Quick Start (newish)</doc>
```

Reply:
Ah, I had no idea that 'newer', 'older', 'newish' would influence the sorting order. I was wondering last night if we should perhaps add the product version in brackets instead of 'newer' etc. (because I considered the adjectives a bit confusing) but if we add the product version in brackets the most recent versions would probably end up further down.

Reply:
Exactly :/ We're using alphabetic sorting, so the oldest version will come out first if you use version numbers. `newer` being sorted before `older` is just a lucky accident that I was using.